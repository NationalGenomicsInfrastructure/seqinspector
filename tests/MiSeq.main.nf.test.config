// Load the basic test config
includeConfig 'nextflow.config'

// Load the correct samplesheet for that test
params {
    input               = params.pipelines_testdata_base_path + 'seqinspector/testdata/MiSeq/samplesheet.csv'
    min_trimmed_reads   = 10
}
