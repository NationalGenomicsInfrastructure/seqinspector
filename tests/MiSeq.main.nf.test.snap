{
    "MiSeq data test": {
        "content": [
            "multiqc_citations.txt:md5,d35df50e9903a96a2b3bce3c1fbc8ad2",
            "multiqc_fastqc.txt:md5,4da95cbe22bce954b82b02ca79979279",
            "multiqc_general_stats.txt:md5,08af93f12fe75b8b95792ea7eef3d81b",
            "multiqc_software_versions.txt:md5,58ba4a50500001b1ba72556e1d797c89",
            "multiqc_citations.txt:md5,d35df50e9903a96a2b3bce3c1fbc8ad2",
            "multiqc_fastqc.txt:md5,4da95cbe22bce954b82b02ca79979279",
            "multiqc_general_stats.txt:md5,08af93f12fe75b8b95792ea7eef3d81b",
            "multiqc_software_versions.txt:md5,58ba4a50500001b1ba72556e1d797c89",
            "multiqc_citations.txt:md5,d35df50e9903a96a2b3bce3c1fbc8ad2",
            "multiqc_fastqc.txt:md5,4da95cbe22bce954b82b02ca79979279",
            "multiqc_general_stats.txt:md5,08af93f12fe75b8b95792ea7eef3d81b",
            "multiqc_software_versions.txt:md5,58ba4a50500001b1ba72556e1d797c89"
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.2"
        },
        "timestamp": "2024-06-27T15:15:28.557909"
    }
}